.github/workflows/pyopenms-wheels.yml

```diff
@@ -338,15 +358,15 @@ jobs:
               # auditwheel now adds the libraries to the root pyopenms.libs directory
               yum install -y zip
               pushd wheelhouse
-                for whl in pyopenms*.whl
+              for whl in pyopenms*.whl
                 do
                   mkdir -p fix
                   cp $whl fix/fix.zip
                   pushd fix
-                    unzip fix.zip
-                    /usr/bin/cp /lib64/libssl.so.10  pyopenms.libs/libssl-*
-                    /usr/bin/cp /lib64/libcrypto.so.10  pyopenms.libs/libcrypto-*
-                    zip -r ../$whl pyopenms/
+                  unzip fix.zip
+                    #/usr/bin/cp /lib64/libssl.so.10  pyopenms.libs/libssl-*
+                    #/usr/bin/cp /lib64/libcrypto.so.10  pyopenms.libs/libcrypto-*
+                  zip -r ../$whl pyopenms/
                   popd
                   rm -rf fix
                 done
```

Comment:
I wonder if LL 352-373 are needed at all now. It seems like without the copying of the crypto libs, it just unzips and zips the wheels again.
I also wonder if we would realize the absence of the libssl immediately during loading (or only when using algorithms that communicate, e.g. MascotOnlineAdapter). Probably during load.. so this is fine.

Reply:
I think they should not be needed. I will remove them. 
I think it should fail during load. The other thing is that there is now a `libssl` & `libcrypto` in the `pyopenms.libs ` directory, so that seems fine.